docs/source/index.rst

```diff
@@ -125,6 +125,7 @@ Documentation
    :caption: User Guides
```

Comment:
For docs, we should update the minimal permissions required for ports in our doc: https://skypilot.readthedocs.io/en/latest/cloud-setup/cloud-permissions/aws.html#user-account

Reply:
@cblmemo - if we do not plan on addressing this comment in this PR, should we file an issue? (ok to do that if you think this will take time to figure out and add)